jenkins-integration/scripts/foss_install.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -x
+
+export GENCONFIG_LAYOUT="${GENCONFIG_LAYOUT:-centos6-64ma}"
+export BEAKER_TESTSUITE="${BEAKER_TESTSUITE:-beaker/install/foss/}"
+export BEAKER_KEYFILE="~/.ssh/id_rsa-acceptance"
+export BEAKER_HELPER="beaker/helper.rb"
+export BEAKER_INSTALL_TYPE="foss"
```

Comment:
@nwolfe Ahh, ok, thanks for the info
Tested it out on my end and it works fine with aio